src/XMakeTasks/UnitTests/project.json

```diff
@@ -1,10 +1,11 @@
 ﻿{
   "dependencies": {
-    "xunit": "2.1.0",
     "microsoft.xunit.netcore.extensions": "1.0.0-prerelease-00629-09",
+    "System.Security.Principal.Windows": "4.3.0-beta-24430-01",
```

Comment:
Can you use the `4.0.0` version instead? We're trying to stick to released packages where possible.